pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -148,6 +151,12 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
 	server.PodInfraContainerImage = imageTemplate.ExpandOrDie("pod")
 	server.CPUCFSQuota = true // enable cpu cfs quota enforcement by default
 
+	if dockerClient, _, err := docker.NewHelper().GetClient(); err == nil {
```

Comment:
FYI, we wanted to do this upstream as part of the Kubelet initialization process, but upstream rejected modifying the Kubelet to do this, and suggested instead that we modify the external bootstrapping code (salt). We are operating under the assumption that it will take significant bake-in time upstream to demonstrate that parallel pulls are safe again (we are optimistic/confident that the Docker 1.9 changes have fixed the concurrency issues), so we're making the change here in Origin instead.